test/tests/functional/pbs_node_rampdown.py

```diff
@@ -6634,8 +6634,12 @@ def test_release_job_array(self):
                  "1:ncpus=2:mem=2097152kb"
         newsel_esc = newsel.replace("+", "\+")
         new_exec_host = self.job1_exec_host
-        new_exec_host_esc = self.job1_exec_host.replace(
-            "*", "\*").replace("[", "\[").replace("]", "\]").replace("+", "\+")
+
+        # Below variable is being used for the accounting log match
+        # which is currently blocked on PTL bug.
```

Comment:
Please name the PTL bug (PP ticket) so when it is fixed, we know to uncomment the code.